### PR TITLE
Add Brazilian dataset

### LINKED ISSUE #97: Add Brazil dataset
- [x] Create sharepoint folder and upload
- [x] Check processing and create dataset files
- [x] Update fetch_data
- [x] Check cleaning code is OK

### PR DESCRIPTION
**Pull request by tarensanders**
Closes #97 

Creating the PR while the full pipeline reruns to make sure nothing broken. I'll mark when it's ready to review.

I had to update a bunch of packages to get this to work. Probably not the worst thing to use the newer versions anyway, but apologies that you'll need to reinstall some packages.

Don't forget to rerun `fetch_data()` to add the dataset before running the pipeline.

### CODE REVIEW

**Comment by tarensanders** [comment]
We gotta cull some images... 96 changed files from rerunning the pipeline.
image

File: .Rprofile [hunk 1]
```diff
@@ -1,4 +1,2 @@
 source("renv/activate.R")
-library(targets)
-targets::tar_load_globals()
```
Comment (conig) [inline .Rprofile:3]: Was this causing problems, or just not necessary?

Comment (tarensanders) [reply .Rprofile:3]: I put the rationale under commit [6548eb0](https://github.com/Motivation-and-Behaviour/sleepIPD_analysis/pull/98/commits/6548eb0866de8609880f23ec3f8329976b5b2509)

TL;DR - it's being loaded by every worker, the docs [explicitly say](https://docs.ropensci.org/targets/reference/tar_load_globals.html#storage-access) not to do this.

Comment (conig) [reply .Rprofile:3]: Ahh problematic, thanks (DR). Ideally there would be a way to get it to load when not executing the pipeline to make dev work seamless, but can see not good to mess with pipeline environments.

**Review by conig — APPROVED** [review]
Looks good!
